client/app/scripts/reducers/root.js

```diff
@@ -84,6 +84,7 @@ export const initialState = makeMap({
   viewport: makeMap(),
   websocketClosed: false,
   zoomCache: makeMap(),
+  serviceImages: makeMap({ fetching: false })
```

Comment:
1. This is just an idea, but I think it might be cleaner to keep the whole the *service* state in one `service` namespace vs. having specific fields (like `serviceImages`). That would make a clear cut in the state tree between the internal and Weave Cloud parts as we continue adding more integrations.
2. I think both `fetching` and `errors` states should be used *per-service* instead of globally (if I already fetched the images for the service A and I'm about to fetch them for service B, I don't want the node details of A to switch back to the fetching state).

To sum up, I'd suggest sort of the following structure: `service -> images -> [serviceId] -> { isFetching, errors, data }`. What do you think?